src/for-redux-swr/SpreadoSetupImpl.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint react-hooks/rules-of-hooks: "off" */
+import {Store} from 'redux';
+import {SpreadoSetup} from '..';
+import {SpreadoSetupOptions} from '../core';
+import {getSpreadIn, setSpreadOut, useSpreadIn, useSpreadOut} from '../redux';
+
+export const EMPTY_BODY_MSG = 'In development';
+
+interface Options extends SpreadoSetupOptions {
+  store: Store;
+}
+
+export class SpreadoSetupBetweenReduxSwr implements SpreadoSetup {
+  private options: Options;
+  private useSpreadOutCounter: Record<string, number> = {};
+
+  constructor(options: Options) {
+    this.options = options;
+  }
+
+  // /* eslint no-unused-vars: "off", @typescript-eslint/no-unused-vars: "off" */
+  // useOnline<T>(...params: UseOnlineParams<T>): UseOnlineResult<T> {
```

Comment:
[Spreado](https://compass-tech.atlassian.net/wiki/spaces/FELIX/pages/3278307938) doesn't include `useOnline`, let's remove it. 🙂